js/utils.js

```diff
@@ -45,6 +45,71 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     return parseFloat(value);
   }
 
+  function getFilesInfo(options) {
+    var entry = options.entryData;
+    var detailViewFileOptions = _.filter(options.detailViewOptions, function(option) {
+      return option.editable && option.type === 'file';
```

Comment:
Why does it need to be `editable`? And if not, use `_.filter(x, { type: 'filter' })` instead.

```suggestion
      return option.type === 'file';
```